matlab_simulation/05-estimation/runFastSLAM.m

```diff
@@ -0,0 +1,61 @@
+%% Fast SLAM Example
+% Author: Bismaya Sahoo.
+%% Initializations
+clear all; close all; clc;
+% Primary Initializations           
```

Comment:
This program is great and well organized.  Only minor thing is regarding the code style. [Use alignment wherever it enhances readability](https://sites.google.com/site/matlabstyleguidelines/layout-documentation/white-space#TOC-Use-alignment-wherever-it-enhances-readability)